plugins/ipqualityscore/icon_ipqualityscore/util/api.py

```diff
@@ -0,0 +1,49 @@
+import json
+from logging import Logger
+from requests import Response, session
+from requests.exceptions import ConnectTimeout, HTTPError, InvalidURL, ProxyError
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+BASE_URL = "https://ipqualityscore.com/api/json"
+IP_ENDPOINT = BASE_URL + "/ip"
+EMAIL_ENDPOINT = BASE_URL + "/email"
+URL_ENDPOINT = BASE_URL + "/url"
+PHONE_ENDPOINT = BASE_URL + "/phone"
+
+
+class IPQSClient:
+    def __init__(self, api_key, logger: Logger):
+        self._headers = {"IPQS-KEY": api_key, "Content-Type": "application/json"}
+        self._logger = logger
+        self.session = session()
+
+    def ipqs_lookup(self, url: str, ad_params: dict) -> dict:
+        try:
+            response = self.session.get(url, headers=self._headers, params=ad_params).json()
+            if str(response["success"]) != "True":

```

Comment:
To remove case sensitivity on that response 😄 
```suggestion
            if str(response.get("success", "")).lower() != "true":

```